catalog/config-controller-monitoring/prometheus-instance.yaml

```diff
@@ -95,4 +80,53 @@ spec:
     targetPort: 9090
   selector:
     app: prometheus
-    prometheus: config-controller
\ No newline at end of file
+    prometheus: config-controller
+---
+# Promethus UI
+# https://cloud.google.com/stackdriver/docs/managed-prometheus/query#promui-deploy
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: frontend
+  namespace: config-controller-monitoring
+spec:
+  replicas: 2
+  selector:
+    matchLabels:
+      app: frontend
+  template:
+    metadata:
+      labels:
+        app: frontend
+    spec:
+      serviceAccountName: prometheus-config-controller
+      containers:
+      - name: frontend
+        image: "gke.gcr.io/prometheus-engine/frontend:v0.1.1-gke.0"
```

Comment:
is this required to submit metrics? or is this just a UI?

Reply:
Just the UI. Noting the UI is also required for Grafana integration so its a nice to have.

Reply:
I guess if we need for Grafana integration, we can pull it in for now, and make a decision on how necessary it is for the alpha.